applications/crossbar/doc/tasks.md

```diff
@@ -469,15 +483,17 @@ curl -v -X PATCH \
 }
 ```
 
+#### Retrieve a task's CSVs
 
-#### Retrieve a task's input CSV
+When you `GET /v2/accounts/{ACCOUNT_ID}/tasks/{TASK_ID}`, the JSON will include a `"csvs":[...]" array with input and output CSVs as appropriate. Use the name(s) in the array to specify which you would like to receive.
 
-> GET /v2/accounts/{ACCOUNT_ID}/tasks/{TASK_ID}/input
+> GET /v2/accounts/{ACCOUNT_ID}/tasks/{TASK_ID}
```

Comment:
While you are free to ADD documentation, you CANNOT remove documentation for EXISTING endpoints.